tools/clang/tools/dxclib/dxc.cpp

```diff
@@ -808,6 +811,51 @@ void DxcContext::Recompile(IDxcBlob *pSource, IDxcLibrary *pLibrary,
   *ppCompileResult = pResult.Detach();
 }
 
+int DxcContext::MaybeRunExternalValidatorAndPrintValidationOutput(
```

Comment:
```suggestion
HRESULT DxcContext::MaybeRunExternalValidatorAndPrintValidationOutput(
```

Reply:
Fair enough, there's some really questionable pre-existing stuff going on in here that we're probably not going to be able tackle in this change.